.licenserc.yaml

```diff
@@ -5,6 +5,12 @@ header:
     content: |
       Copyright [year] [owner]
       See LICENSE file for licensing details.
+  license:
```

Comment:
The format doesn't match https://github.com/canonical/operator-workflows/blob/main/.github/files/.licenserc.yaml